src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+
+        int supply = 0;
+        int buy = 0;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] str = value.split(",");
+                if (str[0].equals("supply")) {
+                    supply += Integer.parseInt(str[1]);
+                } else if (str[0].equals("buy")) {
+                    buy += Integer.parseInt(str[1]);
+                }
+                value = bufferedReader.readLine();
+            }
+
+        } catch (Exception e) {
```

Comment:
The catch block is catching a generic Exception. It's better to catch specific exceptions to handle different error cases with more precision.